package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@folio/myprofile",
+  "version": "0.1.0",
+  "description": "My profile",
+  "repository": "folio-org/ui-myprofile",
+  "publishConfig": {
+    "registry": "https://repository.folio.org/repository/npm-folio/"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "stripes": {
+    "type": "settings",
+    "displayName": "ui-myprofile.meta.title",
+    "route": "/myprofile"
+  },
+  "scripts": {
+    "babel-eslint": "^8.0.3",
+    "lint": "eslint *.js || true",
```

Comment:
Why not `eslint .`? That will recurse into subdirectories, finding all the JavaScript files.

Reply:
Oh, and we shouldn't be using `|| true`, it lies to Jenkins. I guess you copied this from somewhere? If you can remember where, let me know and I'll fix that, too.